data-shaping/output/output.py

```diff
@@ -561,7 +561,7 @@ def transfer_trainees(path, sheets):
                 hcpc_counter_signoff_required=row['RGHCSREQ'],
                 hcpc_signoff_name=row['RGHCNM'],
                 hcpc_signoff_number=row['RGHCNUM'],
-                ahcs_registration=row['RGAHCS'],
+                ahcs_registration=row.get('RGAHCS', 0),
```

Comment:
Hey @aunroel, it would be great to have some comments here on why we're supplying a default here?

Reply:
this is related to https://github.com/torchbox/NSHCS-TMS/pull/7

attempting to access non-existent column (which is the case for `RGAHCS` results in a script error

reason for defaulting to `0` here is simple - it's a numeric field and OutSystem defaults to `0` as null identifiers under the hood anyway.

so this is a pure workaround

Reply:
Cool, makes sense.